SetupDataPkg/Docs/PlatformIntegration/SoftwareComponentOverview.md

```diff
@@ -22,6 +24,7 @@ For more general background about the steps necessary to integrate the configura
 | Driver | Location |
 | ---| ---|
 | ConfDataSettingProvider | SetupDataPkg/ConfDataSettingProvider/ConfDataSettingProvider.inf |
+| ConfProfileMgrDxe | SetupDataPkg/ConfProfileMgrDxe/ConfProfileMgrDxe.inf |
 
 ## UEFI Applications
 

```

Comment:
Missing ActiveProfileSelectorLibNull lib.

Reply:
Probably make that as a separate library section.

Reply:
Will add.